package.json

```diff
@@ -86,6 +86,7 @@
     "@googleapis/sqladmin": "^15.0.0",
     "gaxios": "^6.1.1",
     "google-auth-library": "^9.2.0",
+    "googleapis-common": "^7.1.0",
```

Comment:
this dep is a transitive dep of `@googleapis/sqladmin` but its pinned to `^v7.0.0`.  So not sure for node deps if its best for us to add it here or try to get it bumped in sqladmin?